### PR TITLE
Trying an automatic fix tool. This may fix #42 and many more.

### LINKED ISSUE #42: Clean whitespace errors from CodeSniffer
The latest run of PHP_CodeSniffer came back with about 2,000 instances of whitespace problems (trailing whitespace at the end of lines, using spaces rather than tabs for indenting, etc). This is a third of all the errors reported by our code analysis. 

It is also a very low-hanging fruit to fix, so is a good candidate for something to do in a sprint light with other work.

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This is more of an experiment than anything, as I'm trying out a possibility for automated code _fixing_ rather than just analysis.

Signed-off-by: Matt Bernhardt <mjbernha@mit.edu>

### CODE REVIEW

File: archive.php [hunk 1]
```diff
@@ -35,15 +35,16 @@
...
+            </header><!-- .archive-header -->
 
 			<?php
 			/* Start the Loop */
 			while ( have_posts() ) : the_post();
 
-				/* Include the post format-specific template for the content. If you want to
-				 * this in a child theme then include a file called called content-___.php
-				 * (where ___ is the post format) and that will be used instead.
+				/*
```
Comment (frrrances) [inline archive.php]: This one ended up a little funny.

File: category.php [hunk 2]
```diff
@@ -13,25 +13,26 @@
...
+            </header><!-- .archive-header -->
 
 			<?php
 			/* Start the Loop */
 			while ( have_posts() ) : the_post();
 
-				/* Include the post format-specific template for the content. If you want to
-				 * this in a child theme then include a file called called content-___.php
-				 * (where ___ is the post format) and that will be used instead.
+				/*
```
Comment (frrrances) [inline category.php]: Same funniness here.

Comment (matt-bernhardt) [inline category.php]: Yeah, I'm seeing the same sort of weirdness in the hours library, for example: https://github.com/MITLibraries/MITlibraries-parent/blob/4ec6389289534d86b9059af5939f96915eb9b42b/lib/hours.php#L875

This probably rules out this tool as it is currently configured. I don't want to have to crawl through this large a diff looking for unintended consequences.

**Comment by matt-bernhardt** [comment]
After thinking about this PR a bit more today, both with the UX team and the developer team, I'm not convinced that this is a viable option. It seems a bit too ambitious to try and fix this many problems in an automated fashion - particularly after having found evidence of unintended changes (comment text being changed into interpreted commands).

Because we know this has happened once, in order to approve this change we would need to be very vigilant about reviewing all the changes - which is difficult because at least one file (`/lib/hours.php`) was so significantly changed that its diff is too long to display here.

I'm not abandoning the idea of automated code fixing - just stepping back a bit from doing everything in one fell swoop. There are a number of ways to accomplish this:

* Clean single or small groups of files, rather than all at once.
* Clean single or a few types of problems (just whitespace issues, for example)

**Pull request closed without merging**